internal/host/hostcommands/install_cmd.go

```diff
@@ -235,7 +235,7 @@ func getBootDevice(log logrus.FieldLogger, hwValidator hardware.Validator, host
 
 		return "", newErr
 	}
-	return hostutil.GetDeviceFullName(disks[0].Name), nil
+	return disks[0].ID, nil
```

Comment:
The agent expects to receive a path, not an ID

Reply:
The id is a path....